polaris-react/src/components/InlineCode/InlineCode.scss

```
@@ -0,0 +1,8 @@
+.Code {
+  display: inline-block;
```

```suggestion
  display: inline;
```

Just wanted to double check `inline-block was needed. It is sometimes with padding/border.